scripts/e2e-react-oie.sh

```diff
@@ -0,0 +1,58 @@
+
+
+#!/bin/bash -x
+
+source ${OKTA_HOME}/${REPO}/scripts/setup.sh
+
+setup_service java 1.8.222
+setup_service google-chrome-stable 89.0.4389.72-1
+
+export TEST_SUITE_TYPE="junit"
+export TEST_RESULT_FILE_DIR="${REPO}/build2/reports/e2e"
+export CI=true
+export DBUS_SESSION_BUS_ADDRESS=/dev/null
+
+# Configuration
+# Remember to set this while running tests locally 
+export SAMPLE_NAME=@okta/test.app.react-oie
+export ORG_OIE_ENABLED=true 
+export USERNAME=mary@acme.com
+get_secret prod/okta-sdk-vars/password PASSWORD
+get_vault_secret_key devex/auth-js-sdk-vars a18n_api_key A18N_API_KEY
+export FB_USERNAME=js_ekdtypn_user@tfbnw.net
+get_secret prod/okta-sdk-vars/fb_password FB_PASSWORD
+
+# If this script is run as a bacon task, run against trexcloud environment
+if [[ "${BACON_TASK}" == true ]]; then
```

Comment:
instead of repeating this pattern in each script, maybe source a common file?

Reply:
How about keep it in separated files for now, tasks are still handled a bit differently right now (depends on app type), we can generalize the script pattern after getting more knowledge.